virtualizarr/parsers/hdf/hdf.py

```diff
@@ -175,6 +175,12 @@ def _dataset_chunk_manifest(
     if dataset.chunks is None:
         if dsid.get_offset() is None:
             chunk_manifest = ChunkManifest(entries={}, shape=dataset.shape)
+        elif dataset.shape == ():
+            chunk_manifest = ChunkManifest.from_arrays(
+                paths=np.array(filepath, dtype=np.dtypes.StringDType),
```

Comment:
Perhaps the type annotation for `ChunkManifest.from_arrays` could be improved, as I believe that's the basis for the mypy error here, but I'd have to dig into things and understand why it is annotated as it is to know if there might be a better way to do it.

For now, I'd say you should just add a `# type: ignore` comment on this line.

Reply:
Thanks, @chuckwondo!